src/util/format_expr.cpp

```diff
@@ -324,6 +324,34 @@ std::ostream &format_rec(std::ostream &os, const exprt &expr)
         os << ' ' << format(s);
       return os << " }";
     }
+    else if(statement == ID_dead)
+    {
+      return os << "dead " << format(to_code_dead(code).symbol()) << ";";
+    }
+    else if(statement == ID_decl)
+    {
+      const auto &declaration_symb = to_code_decl(code).symbol();
```

Comment:
Suggest explicitly writing `decl` like we do for the corresponding GOTO program instruction